### PR TITLE
dispatching macros

### PR DESCRIPTION
**Pull request by alieus**
## Description & motivation

Dispatching `compare_column_values` and `compare_queries`. Goal is to create shims for these two macros to make them compatible with TSQL

## Checklist
- [X] I have verified that these changes work locally

### CODE REVIEW

**Comment by swanderz** [comment]
@clrcrl friendly bump -- we built a shim package on this branch and it's now working (except for `compare_relation_columns`). we'd like to let dbt-msft folks start using it, and think it'd probably be easier for users to install this package alongside it, rather than @alieus's fork. But if you've got bigger fish to fry, we're not technically blocked as of yet...

### prefer this
```yaml
packages:
   - package: fishtown-analytics/dbt_audit_helper
     version: 0.3.1
   - package: dbt-msft/tsql_utils
     version: 0.7.0
```

### over this
```yaml
packages:
   - package: alieus/dbt_audit_helper
     version: 0.0.1
   - package: dbt-msft/tsql_utils
     version: 0.7.0
```

**Review by clrcrl — CHANGES_REQUESTED** [review]
We fixed the integration tests in #31 — can you revert the changes to `.circleci/config.yml` here?

Thanks!

**Comment by alieus** [comment]
Done, @clrcrl .

File: macros/compare_queries.sql [hunk 1]
```diff
@@ -1,4 +1,8 @@
-{% macro compare_queries(a_query, b_query, primary_key=None) %}
+{% macro compare_queries(a_query, b_query, primary_key=None) -%}
+  {{ return(adapter.dispatch('compare_queries', packages = audit_helper._get_audit_helper_namespaces())(a_query, b_query, primary_key=None)) }}
```
Comment (clrcrl) [inline macros/compare_queries.sql:2]:
```suggestion
  {{ return(adapter.dispatch('compare_queries', packages = audit_helper._get_audit_helper_namespaces())(a_query, b_query, primary_key)) }}
```

**Review by clrcrl — CHANGES_REQUESTED** [review]
one small thing that I missed!

**Review by clrcrl — APPROVED** [review]

**Comment by clrcrl** [comment]
(waiting for test to pass, but need to run to an errand, will merge / cut a new release later today)

**Pull request merged**